src/components/custom-field-input-date/custom-field-input-date.jsx

```diff
@@ -59,6 +59,14 @@ const CustomFieldInputDate = forwardRef(function CustomFieldInputDate(props, ref
     setShouldFocusInput(false);
   }, [shouldFocusInput]);
 
+  useEffect(() => {
+    if (ref && currentValue) {
+      props.onChange({ target: ref.current });
+    } else if (currentValue) {
```

Comment:
I'm not sure we will ever get into a situation when the state value of the component changes but it is not rendered.

Reply:
So I think the user does not have to pass a ref through, that is was this case is for. Not that it is not rendered.